resources/tentative/charts/observable-plot.js

```diff
@@ -0,0 +1,265 @@
+    // have one item == one airport information.
+    /* Array<{state, iata, name, city, index, origin, destination, total}> */
+    const plotData = stateInformationSortedArray.flatMap(({ mostUsedAirports, total, state }) => {
+        const enrichedMostUsedAirports = mostUsedAirports.map(({ iata, name, city }, index) => ({
+            state,
+            iata,
+            name,
+            city,
+            index, // This will be used to have consistent colors.
+            ...flightsByAirport.get(iata),
+        }));
+        enrichedMostUsedAirports.push({
+            state,
+            iata: "Other",
+            total: total - d3Array.sum(mostUsedAirports, ({ iata }) => flightsByAirport.get(iata)?.total),
+            index: enrichedMostUsedAirports.length,
+        });
+        return enrichedMostUsedAirports;
+    });
+
+    preparedData = {
+        airports,
+        flights,
+        flightsByAirport,
+        byAirport,
+        stateInformationSortedArray,
+        stateSortedArray,
+        statesWithMostFlights,
+        plotData,
+    };
+}
+
+function parseAirportsInformation() {
+    return {
+        airports: csvParse(airportsString),
+        flights: csvParse(flightsString),
+    };
+}
+
+function groupFlightsByAirports(flights) {
+    const flightsByAirport = new Map();
+
+    for (const { origin, destination, count } of flights) {
+        const infoForOriginAirport = flightsByAirport.get(origin) ?? { origin: 0, destination: 0, total: 0 };
+        const intCount = Number(count);
+        if (Number.isNaN(intCount)) {
+            console.error(`Couldn't convert ${count} to number.`);
+            continue;
+        }
+        infoForOriginAirport.origin += intCount;
+        infoForOriginAirport.total += intCount;
+        flightsByAirport.set(origin, infoForOriginAirport);
+        const infoForDestinationAirport = flightsByAirport.get(destination) ?? { origin: 0, destination: 0, total: 0 };
+        infoForDestinationAirport.destination += intCount;
+        infoForDestinationAirport.total += intCount;
+        flightsByAirport.set(destination, infoForDestinationAirport);
+    }
+
+    return flightsByAirport;
+}
+
+function isReady() {
+    return !!preparedData;
+}
+
+function addStackedBars() {
+    if (!isReady())
+        throw new Error("Please prepare the data first.");
+
+    const options = {
+        ...DEFAULT_OPTIONS,
+        color: { type: "categorical" },
+        x: {
+            domain: preparedData.stateSortedArray,
+        },
+        y: { grid: true, tickFormat: "~s" },
+        marks: [
+            // stacked bars
+            Plot.barY(preparedData.plotData, {
+                x: "state",
+                y: "total",
+                fill: "index",
+                title: (d) => `${d.iata === "Other" ? "Other" : `${d.name}, ${d.city} (${d.iata})`}\n${d3Format(",")(d.total)} flights`,
+            }),
+            // labels
```

Comment:
This comment seems unnecessary? The code below is pretty self-explanatory.

Reply:
Well, _I_ don't find this self-explanatory as somebody who's not used to this much. I'd prefer to keep the comments if that's OK. I can expand them a bit to make them more useful.

Reply:
I'd strongly prefer not having any "what" comments as is the case with the rest of WebKit codebase.

Reply:
I removed all the comments now.